packages/pn-personagiuridica-webapp/src/components/Contacts/ContactsSummaryCards.tsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { AddCircle, Verified, WarningOutlined } from '@mui/icons-material';
+import { Card, Stack, Typography } from '@mui/material';
+
+import { AddressType, ChannelType, DigitalAddress } from '../../models/contacts';
+import { useAppSelector } from '../../redux/hooks';
+
+type ContactsSummaryCardProps = {
+  contacts: Array<DigitalAddress>;
+  isSercQEnabled?: boolean;
+  addressType?: AddressType;
+};
+
+const ContactsSummaryCard: React.FC<ContactsSummaryCardProps> = ({
+  contacts,
+  isSercQEnabled,
+  addressType,
+}) => {
+  const { t } = useTranslation('recapiti');
+  const hasAddress = contacts.length > 0;
+  const isCourtesyCard = addressType === AddressType.COURTESY;
+  const title = isCourtesyCard ? 'summary-card.courtesy-title' : 'summary-card.legal-title';
+
+  const getIcon = () => {
+    if (!hasAddress) {
+      if (isCourtesyCard && isSercQEnabled) {
+        return <WarningOutlined color="warning" data-testid="warningIcon" />;
+      }
+      return (
+        <AddCircle
+          color="primary"
+          data-testid="addIcon"
+          aria-label="summary-card.add-icon-arialabel"
+        />
+      );
+    }
+
+    return <Verified color="success" data-testid="verifiedIcon" />;
+  };
+
+  const getDescription = () => {
+    if (!hasAddress) {
+      return t('summary-card.no-address');
+    }
+
+    return contacts
+      .filter((contact) => contact.senderId === 'default')
+      .map((contact) => t(`summary-card.${contact.channelType}`))
+      .join(', ');
+  };
+
+  const goToSection = () => {
+    const sectionId = isCourtesyCard ? 'courtesyContactsSection' : 'legalContactsSection';
+
+    document.getElementById(sectionId)?.scrollIntoView({ behavior: 'smooth' });
+  };
+
+  return (
+    <Card
+      sx={{
+        p: 2,
+        width: { xs: '100%', lg: '185px' },
+        cursor: 'pointer',
+        '&:hover, &:active': { bgcolor: '#F6F7F8' },
+      }}
+      onClick={goToSection}
+      data-testid={isCourtesyCard ? 'courtesyContactsCard' : 'legalContactsCard'}
+    >
+      {getIcon()}
+      <Typography variant="body2" fontWeight={700} mt={0.5} mb={1} data-testid="cardTitle">
+        {t(title)}
+      </Typography>
+      <Typography variant="body2" data-testid="cardDescription">
+        {getDescription()}
+      </Typography>
+    </Card>
+  );
+};
+
+const ContactsSummaryCards: React.FC = () => {
+  const legalAddresses = useAppSelector((state) =>
```

Comment:
use available redux selectors to getting legalAddresses, courtesyAddresses and isSercQEnabled in one shot. check around the code how to do that